client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import SingleAppContainer from './SingleAppContainer.js';
+import {topFree} from './mockData.js';
+import {topGrossing} from './mockData.js';
+import {topPaid} from './mockData.js';
+import {trending} from './mockData.js';
+import './TopChartsStyle.css';
+import '../styles.css';
+ 
+class TopCharts extends Component {
+  render() {
+    const makeColumn = item => (
+        <div key={item.uniqueID}>
+            <SingleAppContainer
+            listNum={item.listNum}
+            url={item.url}
+            title={item.title}
+            category={item.category}
+            rating={item.rating}
+            />
+        </div>
+    );
+
+    return (
+      <div className="col-12 bg-light mx-auto text-center">
+        <div class="section-header">
+          <h4>Top Charts</h4>
```

Comment:
this look like arbitrary headings. they should following the h1, h2, h3, etc. flow.